### DISCUSSION TITLE
Treat Common line as a level

### DISCUSSION THREAD

**Discussion by ribber** [vslinko/obsidian-zoom#7] [Ideas]
For example

```md
## A Title

what is s Java
- Java is a programming language
- Java is owned by Oracle
```

will Treat as

```md
- A Title
  - what is s Java
    - Java is a programming language
    - Java is owned by Oracle
```

So，every Markdown document can transfer to a outline

![image](https://user-images.githubusercontent.com/5986011/119256630-2c06f580-bbf4-11eb-9564-549134fef4cd.png)

click the "Java is a programming language" line will display as below

![image](https://user-images.githubusercontent.com/5986011/119256649-48a32d80-bbf4-11eb-8930-5dd3f773c4d0.png)

hope

```
202105231824 > what is s Java

- Java is a programming language
```

Thank you

**Comment by vslinko** [comment]
Discussion was transferred from https://github.com/vslinko/obsidian-outliner to https://github.com/vslinko/obsidian-zoom

**Reply by ribber** [reply]
ok，thank you
